src/components/Heading.js

```diff
@@ -0,0 +1,24 @@
+import  Grid from "@mui/material/Grid"
+import NoteAltIcon from '@mui/icons-material/NoteAlt';
+import Box from '@mui/material/Box'
+import { Typography } from "@mui/material";
+
+const Heading = () => {
+  return (
+    <div>
+     <Grid container spacing={0} sx={{backgroundColor: '#21b5b4', height: 80}}>
+  <Box sx={{
+        display: "flex",
+        alignItems:"center",
+    }}>
+    <Typography variant='h3' component='h1' marginLeft = {1} color = {'white'}>
```

Comment:
Keep an eye on spacing
marginLeft={1} color={'white'}

Also, I believe this will work fine, `color='white'`